arch/arm64/src/common/arm64_syscall.c

```diff
@@ -436,11 +436,31 @@ int arm64_syscall(uint64_t *regs)
 
           if (rtcb->xcp.kstack != NULL)
             {
-              DEBUGASSERT(rtcb->xcp.kstkptr == NULL &&
-                          rtcb->xcp.ustkptr != NULL);
+              uint32_t usp;
```

Comment:
change to uint64_t? uint32_t isn't enough for arm64.

Reply:
Ofc, copy&paste error. I will fix tomorrow